client/src/helpers/APICalls/createStripeCustomer.ts

```diff
@@ -0,0 +1,18 @@
+import { StripeCustomer } from '../../interface/Payment';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const stripeCustomer = async (): Promise<StripeCustomer> => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({}),
```

Comment:
If there's no request body to include, please feel free to omit this field.

Reply:
Will remove it.

Reply:
successfully implemented in commit 8340f128b930c04eb81e229095baa2623bda7942